README.md

```diff
@@ -1 +1,68 @@
-# Hello-Microverse
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> In this project i have to setup HTML, CSS, and linters file.
+
+![screenshot](Hello_Microverse.png)
+
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/Dipeshtwis/Hello-Microverse/feature/hello-microverse/index.html)
+
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine.
+
+## How To Use 🔧
+
+From your command line, first clone the project:
+
+```bash
+# Clone this repository
+$ git clone https://github.com/Dipeshtwis/Hello-Microverse.git
+
+# Go into the repository
+$ cd Hello-Microverse
+
+```
+
+Open `index.html` to see the GUI of the project.
+
+
+## Authors
+
+👤 **Dipesh Kumar**
+
+- [Github](@Dipeshtwis)
+- [Twitter](@97deepeshkumar)
+- [Linkedin](https://www.linkedin.com/in/dipesh-kumar-b6ab88134/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [optional] In order to make your repository professional, it is mandatory to make sure that all of the information presented on readme file are accurate, so because of that I would suggest you fix the following links which are on readme file but they are broken: _license link, issues link, twitter link, github link_

Reply:
Done